torchvision/datasets/video_utils.py

```diff
@@ -64,13 +64,15 @@ class VideoClips(object):
         frame_rate (int, optional): if specified, it will resample the video
             so that it has `frame_rate`, and then the clips will be defined
             on the resampled video
+        num_workers (int): how many subprocesses to use for data loading.
+        0 means that the data will be loaded in the main process. (default: 0)
```

Comment:
Can you add a space in here, as with the other fields, so that it gets rendered properly?